articles/an-overview-of-database-testing/index.md

```diff
@@ -0,0 +1,83 @@
+A database is a crucial element in software applications because it enables users to store and retrieve data. Nowadays, complex applications requiring huge data have increased the need for secure and robust databases. These databases need to be tested to ensure that the huge data stored in them is handled seamlessly. Database testing is important because it verifies the quality, accuracy, and security of data in databases. It establishes whether there are errors in database systems to enhance the consistency, performance, and reliability of these systems. 
+
+In this article, we look at the fundamental aspects of database testing. These include the definition of database testing, its importance, and types of database testing. The article has also provided an overview of the main stages of database testing and popular testing tools.
+
+### What is Database Testing?
+Understanding the concept of database testing requires knowledge of the terms 'data' and 'databases'. Data is a term used for describing facts, information, or statistics collected for various purposes. A database is a collection of data that is organized and stored for access (usually using a computer system). Some of the examples of databases include MySQL, Oracle, dBASE, and FileMaker. 
+
+Database testing is the process of validating data existing in a database by substantiating the elements that affect the data and other related functionalities. It can also be referred to as back-end testing. Some of the tests conducted during database testing include data validity checks, data integrity tests, tests for procedures, triggers, performance checks, and database functions.
+
+This validation process is used in an online analytical processing (OLAP) system. Some of the tools that can be used for database testing include Selenium and QuickTest Professional (QTP). 

```

Comment:
> This validation process is used in an online analytical processing (OLAP) system.

It would be great if you can elaborate on what an OLAP system is (In 1-2 lines)

Reply:
Done.